main.yml

```diff
@@ -1,9 +1,14 @@
 ---
 - hosts: 127.0.0.1
   connection: local
+  vars_prompt:
```

Comment:
I like this addition! It's convenient for new installs, when all the user has is a command prompt. This way they don't have to open a config file, edit it to set these variables, before running dot-ansible. 

However, the majority of the time, this playbook is called in a system where it's already installed. It's being run to update the system. It's annoying and unnecessary to prompt the user for this information each time.

My current idea for user specific information is an inventory file, named after the user, that can be loaded at run time. Could we maybe use the existence of that file, or better yet checking if the vars themselves are already set, to determine if we should prompt the user?